search/text_search.cpp

```diff
@@ -23,20 +51,45 @@ int main() {
     if (paragraph.empty()) {
         std::cout << "\nThe paragraph is empty" << std::endl;
     } else {
+        int ch = 0;
         while (true) {
             std::string word;
             std::cout << "Please enter the word you are searching for: ";
             std::getline(std::cin, word);
-            std::cout << "Hello, your word is " << word << "!\n";
-            if (paragraph.find(word) == std::string::npos) {
-                std::cout << word << " does not exist in the sentence"
-                          << std::endl;
+            std::cout << "Ignore case? 1 = Yes, 0 = No" << std::endl;
+            std::cin >> ch;
+            if (ch == 1) {
+                std::string lowerCase = lower(
+                    paragraph);  // convert std::string paragraph to lowercase
+                                 // and store it in std::string lowerCase
+                std::string lowerCaseWord =
+                    lower(word);  // convert std::string paragraph to lowercase
+                                  // and store it in std::string lowerCase
+
+                std::cout << "Hello, your word is " << word << "!\n";
+                if (lowerCase.find(lowerCaseWord) == std::string::npos) {
+                    std::cout << word << " does not exist in the sentence"
+                              << std::endl;
+                } else {
+                    std::cout << "The word " << word
+                              << " is now found at location "
+                              << lowerCase.find(lowerCaseWord) << std::endl
+                              << std::endl;
+                }
             } else {
-                std::cout << "The word " << word << " is now found at location "
-                          << paragraph.find(word) << std::endl
-                          << std::endl;
+                std::cout << "Hello, your word is " << word << "!\n";
+                if (paragraph.find(word) == std::string::npos) {
+                    std::cout << word << " does not exist in the sentence"
+                              << std::endl;
+                } else {
+                    std::cout << "The word " << word
+                              << " is now found at location "
+                              << paragraph.find(word) << std::endl

```

Comment:
Please add a note that using `Ctrl+C` will exit the program. 🙂

Reply:
Yes sure! 😃